pages/common/mkdir.md

```diff
@@ -9,3 +9,7 @@
 - Create directories recursively (useful for creating nested dirs):
 
 `mkdir -p {{path}}`
+
+- Create multiple directories recursively (note the curly braces, marking directory set):
+
+`mkdir -p dir1/dir2/{dir3/{dir4,dir5},dir6,dir7}`
```

Comment:
I meant it feels a bit too complex for a project that aims to provide short lists of the most common uses of a command, not an exhaustive listing of what can be done with it. But I'm not opposed to adding this, just raising the question so that we make that decision explicitly. I'll let others chime in :)